_posts/hi/newsletters/2022-07-27-newsletter.md

```diff
@@ -0,0 +1,138 @@
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [<!--why-do-invalid-signatures-in-op-checksigadd-not-push-to-the-stack--> `OP_CHECKSIGADD` में अमान्य हस्ताक्षर Stack पर क्यों नहीं धकेलते?]({{bse}}114446)
+  Chris Stewart पूछते हैं कि "यदि कोई अमान्य हस्ताक्षर पाया जाता है, तो दुभाषिया जारी रखने के बजाय
+  निष्पादन में विफल रहता है"। Peter Wuille बताते हैं कि यह व्यवहार, जिसे BIP340-342
+  में परिभाषित किया गया है, भविष्य में [schnorr हस्ताक्षर][topic schnorr signatures] के बैच सत्यापन का
+  समर्थन करने के लिए डिज़ाइन किया गया है। Andrew Chow व्यवहार के लिए एक अतिरिक्त कारण देते हैं, यह देखते हुए
+  कि इस दृष्टिकोण से कुछ लचीलापन संबंधी चिंताओं को भी कम किया जाता है।
+
+- [<!--what-are-packages-in-bitcoin-core-and-what-is-their-use-case-->Bitcoin Core में पैकेज क्या हैं और उनका उपयोग मामला क्या है?]({{bse}}114305)
+  Antoine Poinsot बताते हैं [पैकेज][bitcoin docs packages] (संबंधित लेनदेन का एक समूह), [पैकेज रिले][topic package relay] से उनका संबंध, और हाल ही में [पैकेज रिले BIP प्रस्ताव][news201 package relay]।
+
+- [<!--how-much-blockspace-would-it-take-to-spend-the-complete-utxo-set-->संपूर्ण UTXO सेट को खर्च करने में कितना ब्लॉकस्पेस लगेगा?]({{bse}}114043)
+  Murch सभी मौजूदा UTXO को समेकित करने के एक काल्पनिक परिदृश्य की पड़ताल करता है।
+  वह प्रत्येक आउटपुट प्रकार के लिए ब्लॉकस्पेस गणना प्रदान करता है और निष्कर्ष निकालता है कि इस प्रक्रिया में लगभग 11,500 ब्लॉक लगेंगे।
+
+- [<!--does-an-uneconomical-output-need-to-be-kept-in-the-utxo-set-->क्या UTXO सेट में एक गैर-आर्थिक आउटपुट रखने की आवश्यकता है?]({{bse}}114493)
+  Stickies-v नोट करता है कि 'OP_RETURN' या अधिकतम स्क्रिप्ट आकार से बड़ी स्क्रिप्ट सहित UTXO को
+  UTXO से हटा दिया जाता है। सेट करें, [अनौपचारिक आउटपुट][topic uneconomical outputs] को
+  हटाने से समस्याएं हो सकती हैं, जिसमें एक हार्ड फोर्क भी शामिल है, जैसा कि Peter Wuille बताते हैं, अगर उन आउटपुट
+  को खर्च किया जाता है।
+
+- [<!--is-there-code-in-libsecp256k1-that-should-be-moved-to-the-bitcoin-core-codebase-->क्या libsecp256k1 में कोड है जिसे Bitcoin Core कोडबेस में ले जाया जाना चाहिए?]({{bse}}114467)
+  Bitcoin Core कोडबेस के क्षेत्रों को संशोधित करने के अन्य प्रयासों के समान [libbitcoinkernel][libbitcoinkernel project]
+  या [प्रक्रिया पृथक्करण][devwiki process separation], Peter Wuille [libsecp256k1][] परियोजना की जिम्मेदारी का एक
+  स्पष्ट क्षेत्र नोट करता है: वह सब कुछ जिसमें निजी या सार्वजनिक कुंजी पर संचालन शामिल है।
+
+- [<!--mining-stale-low-difficulty-blocks-as-a-dos-attack-->एक DoS हमले के रूप में पुराने कम-कठिनाई वाले ब्लॉकों का खनन]({{bse}}114241)
+  Andrew Chow बताते हैं कि [assumevalid][assumevalid notes] और हाल ही में [`nMinimumChainWork`][bitcoin core #9053]
+  कम-कठिनाई के श्रृंखला हमलो को फ़िल्टर करने में मदद करते हैं ।
+
+## रिलीज और रिलीज उम्मीदवार
+
+*लोकप्रिय Bitcoin इन्फ्रास्ट्रक्चर परियोजनाओं के लिए नए रिलीज और रिलीज उम्मीदवार। कृपया नई रिलीज़ में
```

Comment:
There was an extra character/space here causing misformatting of this text. I pushed a commit to fix this.

Reply:
Thanks, @bitschmidty!